### PR TITLE
Fix/tao 1207 number of unanswered (attempt 2)

### PR DESCRIPTION
**Pull request by ssipasseuth**
second attempt to fix the issue : https://oat-sa.atlassian.net/browse/TAO-1207

### CODE REVIEW

**Comment by ssipasseuth** [comment]
requires oat-sa/extension-tao-itemqti#402 and oat-sa/extension-tao-item#132

File: views/js/controller/runtime/testRunner.js [hunk 1]
```diff
@@ -234,6 +234,11 @@ define([
                     unansweredCount=(this.testContext.numberItemsSection - this.testContext.numberCompletedSection),
                     flaggedCount=this.testContext.numberFlaggedSection;
 
+                if( this.isCurrentItemAnswered() ){
+                    unansweredCount--;
+                }
+
+                this.getQtiRunner().updateItemApi();
```
Comment (ssipasseuth) [inline views/js/controller/runtime/testRunner.js]: see with @llecaque 
The goal was to fix the code from @bursonic : I just had the time to add the listener of responsechange and stateready but cannot find a way to properly remove the call to this.getQtiRunner().updateItemApi() to submit the response before leaving the item.

**Comment by krampstudio** [comment]
As this one is part of these merging mess and is already part from develop, I leave to @llecaque for tracking (I don't know whet to do with it right now).

**Comment by llecaque** [comment]
act-test-patch is become dev-act , closing this one

**Pull request closed without merging**